test/types/records/expiring/ex.good

```diff
@@ -0,0 +1,70 @@
+Expiring values for function refExample1EOB (ex.chpl:14)
+  rec (ex.chpl:15) expires at end of block
+Expiring values for function refExample2EOB (ex.chpl:20)
+  rec (ex.chpl:21) expires at end of block
+Expiring values for function borrowExample1EOB (ex.chpl:26)
+  c (ex.chpl:27) expires at end of block
+Expiring values for function borrowExample2EOB (ex.chpl:32)
+  c (ex.chpl:33) expires at end of block
+Expiring values for function borrowExample3EOB (ex.chpl:38)
+  c (ex.chpl:39) expires at end of block
+Expiring values for function borrowExample4EOB (ex.chpl:44)
+  c (ex.chpl:45) expires at end of block
+Expiring values for function borrowExample5EOB (ex.chpl:51)
+  c (ex.chpl:52) expires after last mention
+Expiring values for function recordExample1LM (ex.chpl:62)
+  rec (ex.chpl:63) expires after last mention
+Expiring values for function recordExample2LM (ex.chpl:68)
+  rec (ex.chpl:69) expires after last mention
+Expiring values for function makeR (ex.chpl:74)
+  temp (ex.chpl:75) expires at end of block
+Expiring values for function recordExample3LM (ex.chpl:77)
+  rec (ex.chpl:78) expires after last mention
+Expiring values for function ioExample0LM (ex.chpl:84)
+  tmp (ex.chpl:85) expires after last mention
+Expiring values for function ioExample1LM (ex.chpl:89)
+  tmp (ex.chpl:90) expires after last mention
+  A (ex.chpl:92) expires after last mention
+  B (ex.chpl:93) expires after last mention
+  temp (ex.chpl:93) expires at end of block
+  temp (ex.chpl:95) expires at end of block
+  temp (ex.chpl:96) expires at end of block
+Expiring values for function ownedExample1EOB (ex.chpl:100)
+  temp (ex.chpl:101) expires at end of block
```

Comment:
I would expect it to report on `b` as well.

Reply:
It's focused on variables that have meaningful init/deinit/copy-init. It doesn't report on `b` because when `b` expires is not particularly relevant today (because it's a POD pointer-to-instance).